dom.bs

```diff
@@ -1419,6 +1419,107 @@ that gave folks all the wrong ideas. <a>Events</a> do not represent or cause act
 can only be used to influence an ongoing one.
 
 
+<h2 id=canceling-ongoing-activities>Canceling Ongoing Activities</h3>
+
+Though {{Promise}} objects don't have any built-in cancellation mechanism, many APIs using these
+concepts require cancellation semantics. {{CancellationController}} is meant to support these
+requirements by providing an {{CancellationController/cancel()}} method that toggles the state of a
+corresponding {{CancellationSignal}} object. The API which wishes to support cancellation can accept
+such a {{CancellationSignal}}, and use its state to determine how (not) to proceed.
+
+APIs that rely upon {{CancellationController}} are encouraged to respond to
+{{CancellationController/cancel()}} by rejecting any unsettled {{Promise}} with a
+<dfn exception>CancellationError</dfn> exception.
+
+<div class=note>
+ For example, a hypothetical <code>doAmazingness({ ... })</code> method could accept a
+ {{CancellationSignal}} object in order to support cancellation as follows:
+
+ <pre class=lang-javascript>
+  const controller = new CancellationController();
+  const signal = controller.signal;
+  doAmazingness({ ..., signal })
+    .then(result => ...)
+    .catch(e => ...);
+
+  ...
+
+  controller.cancel();
+  </pre>
+
+ APIs that require more granular control could extend both {{CancellationController}} and
+ {{CancellationSignal}} according to their needs.
+</div>
+
+<h3 id=interface-cancellationcontroller>Interface {{CancellationController}}</h3>
+
+<pre class="idl">
+[Constructor(), Exposed=(Window,Worker)]
+interface CancellationController {
+  readonly attribute CancellationSignal signal;
```

Comment:
Done in https://github.com/whatwg/dom/pull/434/commits/36028dd814d2144acfd4efa88b36b66742208d79